gitbook/about/brief-history.html

```diff
@@ -2187,6 +2200,110 @@
             
 
             
+        </li>
+    
+        <li class="chapter " data-level="1.5.124" data-path="../tables/extended-helmholtz-ellis-accidentals-just-intonation-supplement.html">
+            
+                <a href="../tables/extended-helmholtz-ellis-accidentals-just-intonation-supplement.html">
+            
+                    
+                    Extended Helmholtz-Ellis accidentals (just intonation) supplement
+            
+                </a>
+            
+
+            
+        </li>
+    
+        <li class="chapter " data-level="1.5.125" data-path="../tables/other-accidentals.html">
```

Comment:
I believe these two URLs should be other-accidentals-supplement.html rather than other-accidentals.html.